client/jetpack-cloud/sections/partner-portal/payment-methods-v2/empty-state.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button } from '@automattic/components';
+import { Icon, lock, currencyDollar, postDate } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import CreditCardImg from 'calypso/assets/images/jetpack/credit-cards.png';
```

Comment:
[non-blocker] If possible, we can extract the SVG version of the image for better scalability on different screen sizes. Otherwise, please ignore this comment.

Reply:
I tried the extract `.svg`. But, there seems to be a problem. So, I decided to use `.png`